chemprop/train/cross_validate.py

```diff
@@ -63,6 +64,14 @@ def cross_validate(args: TrainArgs,
     data = get_data(path=args.data_path, args=args, logger=logger, skip_none_targets=True)
     validate_dataset_type(data, dataset_type=args.dataset_type)
     args.features_size = data.features_size()
+    args.atom_features_size = 0
```

Comment:
Can you make `atom_features_size` and `atom_descriptors_size` (and any other new properties that I missed that are not command line arguments) Python properties in `args.py` similar to `features_size`? Your current implementation does work fine, but the nice thing about explicitly defining these variables as properties in the `TrainArgs` class is if you use a Python IDE like PyCharm, then it will then know about that property and can do things like autocomplete or type check any time you use that property.

For example, the changes you would need to make would be to add `self._atom_features_size = None` to the `__init__` of `TrainArgs` and then add:

```python
    @property
    def atom_features_size(self) -> int:
        """The dimensionality of the additional atom-level features."""
        return self._atom_features_size

    @features_size.setter
    def features_size(self, atom_features_size: int) -> None:
        self._atom_features_size = atom_features_size
```